lib/openstax/aws/stack.rb

```diff
@@ -338,6 +338,52 @@ def status
       end
     end
 
+    def status_reason
+      begin
+        aws_stack.stack_status_reason
+      rescue Aws::CloudFormation::Errors::ValidationError => ee
+        case ee.message
+        when /Stack.*does not exist/
+          "DOES_NOT_EXIST"
+        else
+          raise
+        end
+      end
+    end
+
+    def events
+      begin 
+        aws_stack.events()
+      rescue
+        raise "something's wrong sorry"
```

Comment:
Same here, is there a reason why the original error is not shown?

Reply:
while testing, sometimes I would get `cannot get events() of nil` so I added this

Reply:
was `aws_stack` nil in that case?  or otherwise should the response in this case be `[]`?